src/main/python/main.py

```diff
@@ -9,16 +10,72 @@
 
 class AppContext(ApplicationContext):
 
+    def __init__(self, *args, **kwargs):
+        super(AppContext, self).__init__(*args, **kwargs)
+        self.mainapp = MainApp(self)
+
+    @cached_property
+    def dashboard_style(self):
+        return self.load_style("dashboard.qss")
+
+    @cached_property
+    def app_style(self):
+        return self.load_style("app.qss")
+
+    @cached_property
+    def login_style(self):
+        return self.load_style("login.qss")
+
+    @cached_property
+    def credit_history_style(self):
+        return self.load_style("credit_history.qss")
+
+    @cached_property
+    def notification_style(self):
+        return self.load_style("notification.qss")
+
+    @cached_property
+    def add_view_style(self):
+        return self.load_style("add_view.qss")
+
+    @cached_property
+    def controller_style(self):
+        return self.load_style("controller.qss")
+    
+    @cached_property
+    def popup_style(self):
+        return self.load_style("popup.qss")
+
+    @cached_property
+    def jobs_style(self):
+        return self.load_style("jobs.qss")
+
+    @cached_property
+    def settings_style(self):
+        return self.load_style("settings.qss")
+
+    @cached_property
+    def question_style(self):
+        return self.load_style("question.qss")
+
     # def __init__(self):
     #     popup_style = self.get_resource("popup.qss")
     #     resources_add_view_style = self.get_resource("add_view.qss")
     #     resources_controller_style = self.get_resource("controller.qss")
 
     def run(self):
         # Save reference to main application so it's not garbage collected
-        dont_garbage_collect_me = MainApp()
+        # dont_garbage_collect_me = MainApp(self)
         return self.app.exec_()
 
+    def load_style(self, path):
+        try:
+            file_path = self.get_resource(path)
+            with open(file_path) as f:
+                return f.read()
+        except FileNotFoundError:
```

Comment:
If we're still raising a `FileNotFoundError` is this necessary? I feel it would be better to let the original error be raised instead of squashing it with a generic message about the file not being found. Or rather, we could log and then raise the error.

Reply:
Did some research, is it something like:
```python
try:
   ...
except:
   raise
```